src/libcore/num/mod.rs

```diff
@@ -119,6 +119,34 @@ macro_rules! int_impl {
      $add_with_overflow:path,
      $sub_with_overflow:path,
      $mul_with_overflow:path) => {
+
+        // FIXME(#11621): Should be deprecated once CTFE is implemented in favour of
+        // calling the `mem::size_of` function.
+        #[unstable(feature = "num_bits_bytes",
+                   reason = "may want to be an associated function",
```

Comment:
Why? It still may want to be an associated function, just like `std::usize::BYTES`.